src/action/payment.js

```diff
@@ -246,7 +246,10 @@ class PaymentAction {
       }
       this._nav.goPayBitcoinConfirm();
     } catch (err) {
-      this._notification.display({ msg: 'Fee estimation failed!', err });
+      this._notification.display({
+        msg: `Fee estimation failed: ${err.details}`,
+        err,
+      });
```

Comment:
`err.details` has always been `undefined` for other errors thrown on mobile. Did this work in your test?

Reply:
Yes it did, but I can add a null check.